# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../blitz/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur blitz-0.6-orig/blitz/funcs.h blitz-0.6/blitz/funcs.h
--- blitz-0.6-orig/blitz/funcs.h	2008-07-23 05:22:49.000000000 +0000
+++ blitz-0.6/blitz/funcs.h	2008-07-23 05:23:29.000000000 +0000
@@ -44,6 +44,8 @@
  #include <blitz/prettyprint.h>
 #endif
 
+#include <cstdlib>
+
 BZ_NAMESPACE(blitz)
     
 /* Helper functions */
diff -Nur blitz-0.6-orig/blitz/prettyprint.h blitz-0.6/blitz/prettyprint.h
--- blitz-0.6-orig/blitz/prettyprint.h	2008-07-23 05:22:49.000000000 +0000
+++ blitz-0.6/blitz/prettyprint.h	2008-07-23 05:23:45.000000000 +0000
@@ -38,6 +38,8 @@
 #ifndef BZ_PRETTYPRINT_H
 #define BZ_PRETTYPRINT_H
 
+#include <cstdlib>
+
 BZ_NAMESPACE(blitz)
 
 class prettyPrintFormat {
